apps/demo/src/app/05-advanced/advanced-wizard/components/wizard-container.component.scss

```
@@ -34,6 +34,45 @@
+.success-message__details dt {
+  font-size: 0.75rem;
+  font-weight: 600;
+  letter-spacing: 0.02em;
+  text-transform: uppercase;
+  color: #15803d;
+}
+
+.success-message__details dd {
+  margin: 0.125rem 0 0;
+  font-weight: 600;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)